librz/asm/arch/pyc/opcode.h

```diff
@@ -46,6 +46,7 @@ typedef struct {
 	ut8 extended_arg;
 	ut8 have_argument;
 	ut8 bits;
+	ut8 jump_use_instruction_offset;
```

Comment:
I guess this should be a `bool`, right? If so, use bool and true/false instead of 0/1